chemqulacs/qse/qse.py

```diff
@@ -61,7 +61,7 @@ def gen_excitation_operators(self, types="ee", n_excitations=2):
     def build_H(self):
         noperators = len(self.e_op)
         op_mapper = self.vqeci.fermion_qubit_mapping.get_of_operator_mapper(
-            self.vqeci.n_electron, self.vqeci.n_qubit
```

Comment:
I expect that with the current changes, some codes will break or not perform correctly when using an older version of Quri_parts.

There are two types of modifications.

1. Reorder of inputs:
This we can counter with using keyword arguments like:
op_mapper = self.vqeci.fermion_qubit_mapping.get_of_operator_mapper(n_fermions=self.vqeci.n_electron, n_spin_orbitals = self.vqeci.n_qubit)
However, this will make our function calls much longer.

2. Different number/type variables for function calls.
I think this is a much bigger issue. To execute the function correctly, we would need to add version checks before each call. This would make the codes much less readable and add a lot of overhead, but it is, of course, possible. 
We would need to remember that in the future all checks need to be done for both versions.